EIPS/eip-draft_my_own_messages.md

```diff
@@ -0,0 +1,145 @@
+---
+eip: <to be assigned>
+title: My Own Messages (MOM)
+author: Giuseppe Bertone (@Neurone)
+discussions-to: https://github.com/InternetOfPeers/EIPs/issues/1
+status: Draft
+type: Standards Track
+category (*only required for Standard Track): ERC
+created: 2020-08-02
+---
+
+<!--You can leave these HTML comments in your merged EIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new EIPs. Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
+My Own Messages (MOM) is a standard to notarize and share messages using Ethereum, so that contents are verificable and decentralized.
+
+Main advantages:
+- You can send messages to users of your ÐApp or Smart Contract, and they always know it is a voice reliable as the smart contract is.
+- Create your Ethereum account dedicated to your personal messages, say something only once and it can be seen on every social platform (no more reply of the same post/opinion on dozens of sites like Reddit, Twitter, Facebook, Medium, Disqus, and so on...)
+- Small fee to be free: pay just few cents of dollar to notarize your messages, and distribute them with IPFS, Swarm or any other storage you prefer. Because the multihash of the content is notarized, you can always check the integrity of the message you download even from centralized storage services.
+- Finally, you can ask and get tips for your words directly into your wallet.
+
+I know, My Own Messages (MOM) sounds like _mom_. And yes, pun intended :)
+
+## Abstract
+<!--A short (~200 word) description of the technical issue being addressed.-->
+My Own Messages (MOM) is a standard to notarize and share messages using Ethereum. You can give voice to your smart contracts, send messages to the world, create a certified blog with your ideas, and so on.
```

Comment:
This section should include a bit more technical detail about how you achieve that.  You don't want to rewrite the specification here, but give a very high level overview of what this standard achieves and how it achieves it.  Something like "...this is achieved by submitting a self-send message to the blockchain with a specific format and message data attached.